client/components/map/index.jsx

```diff
@@ -32,19 +38,29 @@ const map = ({
     />
     { GeoJSON.data.map(feature => (
       <GeoJson
+        {...active}
         key={feature._id}
         data={feature}
-        style={myStyle}
+        style={getActive(active, feature) ? activeStyle : myStyle}
```

Comment:
You literally just suggested we do it this way.. I had removed the other function.. Guess that commit isn't here yet.